pagopa/io-web-profile
# [IOPID-1618] : Updating OnetTrust CookieBanner Scripts to ver 28.02.2024

## Short description

The purpose of this PR is to update the files and scripts of OneTrust for the cookie banner to the latest drafted version.

## How to test locally

1. **FIRST CASE DEV MODE**

    Insert the following variable into the local environment file
    
    NEXT_PUBLIC_DEV_MODE=true

    Start the project in dev mode and ensure that the cookie banner is displayed

2. **SECOND CASE PROD MODE**

    Build the project (production)
  
    Start the project and ensure that the cookie banner is displayed
